P5/Source/Specs/att.cReferencing.xml

```diff
@@ -16,13 +16,13 @@
       <gloss versionDate="2008-04-06" xml:lang="es">referencia canónica</gloss>
       <gloss versionDate="2008-03-30" xml:lang="fr">référence canonique</gloss>
       <gloss versionDate="2007-11-06" xml:lang="it">riferimento canonico</gloss>
-      <gloss xml:lang="ja" versionDate="2018-12-31">正規化参照</gloss>
+      <gloss xml:lang="ja" versionDate="2018-12-31">基準的形式の参照</gloss>
```

Comment:
The `@versionDate` should probably be updated as well

Reply:
I'm sorry for the mistake. I've solved it.